test/typescript.js

```diff
@@ -1,11 +1,12 @@
 import * as tt from 'typescript-definition-tester'
 
 describe('TypeScript definitions', function () {
+  this.timeout(4000);
```

Comment:
Why?

Reply:
I don't know why, but some times compiler waist much time to process the whole test.